components/common/Button.tsx

```diff
@@ -1,15 +1,16 @@
-import { DEVICE_SIZE } from '@/styles/DeviceSize';
+import { DEVICE_SIZE } from '@/lib/styles/DeviceSize';
 import { ReactNode } from 'react';
 import styled from 'styled-components';
 
 interface Props {
-  type: 'add' | 'login' | 'modal' | 'delete' | 'linkAdd';
+  type?: 'add' | 'login' | 'modal' | 'delete' | 'linkAdd';
   children: ReactNode;
 }
 
 function Button({ type, children }: Props) {
   return (
     <>
+      {typeof type === 'undefined' && <BlueGrd>{children}</BlueGrd>}
```

Comment:
음 이런건 이렇게 체크하는게 나을것같기도해요!
```suggestion
     if(!type) return (<BlueGrd>{children}</BlueGrd>);
     return (
     <>
      {type === 'add' && <AddBtn>{children}</AddBtn>}
      {type === 'login' && <LoginBtn>{children}</LoginBtn>}
      {type === 'modal' && <ModalBtn>{children}</ModalBtn>}
      </>
      )
```